src/components/layer-metadata/layer-metadata.vue

```diff
@@ -0,0 +1,155 @@
+<script setup lang="ts">
+import { formatDate } from '@/services/common/formatting.utils'
+import { useTranslation } from 'i18next-vue'
+import LayerMetadataItem from './layer-metadata-item.vue'
+import { useMetadataStore } from '@/stores/metadata.store'
+import { storeToRefs } from 'pinia'
+import { watch, Ref, ref, onMounted } from 'vue'
+import { layerMetadataService } from '@/services/layer-metadata/layer-metadata.service'
+import { LayerMetadataModel } from '@/services/layer-metadata/layer-metadata.model'
+
+const metadataStore = useMetadataStore()
+const { metadataTreeNode } = storeToRefs(metadataStore)
+const { t, i18next } = useTranslation()
+const layerMetadata: Ref<LayerMetadataModel | undefined> = ref()
+
+watch(metadataTreeNode, async node => {
+  node
```

Comment:
weird syntax

`layerMetadata.value = node ? ... : ...` is more readable.

Reply:
oups